src/main/java/com/epam/tc/hw9/service/Utils.java

```diff
@@ -0,0 +1,40 @@
+package com.epam.tc.hw9.service;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class Utils {
+
+    public static String BASE_URI;
```

Comment:
make them private

Reply:
I can't use access modifier 'private' because in this case I can't use this variables in TrelloApiTests class. Maybe protected ?

Reply:
What construction is used in java to access private variables?

Reply:
getter and setter, I think